### DISCUSSION TITLE
How to use Node-rfc within an CAP-based app in BTP

### DISCUSSION THREAD

**Discussion by eliugonzalez86** [SAP/node-rfc#298] [Q&A] [answered]
Hello everyone,
I found this library and I wonder if I can use it in an CAP-based App deployed on BTP. My biggest problem is creating the sapnwrfc.ini file in the CAP project and the installation of the SAP NWRFC SDK

does anyone have an example or guidance?

thanks a lot

**Comment by bsrdjan** [comment, marked as answer]
Hello @eliugonzalez86,

CAP, BTP and Cloud Connector integration require SAP NW RFC SDK cloud variant and connection parameters supported in that version. The release is not GA (globally available) and can be shipped for use-cases described in point 12 of [Using Connectors](https://wiki.one.int.sap/wiki/display/PIC/UsingConnectors) section "Additional rules and info for NW RFC SDK". There is also the  download link for SAP NW RFC SDK cloud variant.

Please check the info and for example you can reach me via Teams or Email

Best regards,
Srdjan

**Comment by bsrdjan** [comment]
Updated answer: check https://blogs.sap.com/2023/10/26/abap-rfc-connectivity-from-btp-node.js-buildpack/
